stacks/spica/src/bucket/pages/add/add.component.ts

```diff
@@ -42,7 +48,13 @@ export class AddComponent implements OnInit {
       switchMap(params => {
         if (params.rid) {
           return this.bds.findOne(params.id, params.rid, true).pipe(
-            tap(data => (this.data = data)),
+            tap(data => {
+              this.data = data;
+              if (data._schedule) {
+                this.scheduleOn = true;
+                this.scheduleDate = new Date(data._schedule);
```

Comment:
You can get rid of this if statements by converting it to `this.scheduleDate = data._schedule && new Date(data._schedule)` thus we can use undefined state of this property to indicate whether this document is scheduled.